cost_calculator/app.py

```diff
@@ -22,50 +23,81 @@
 app.secret_key = os.urandom(24)
 
 
-def k8s_container_resource_requirements(container):
+def k8s_container_resource_requirements(container) -> Dict[str, int]:
+    """
+    returns dict in format:
+     {"req_cpu": 1, "req_mem": 1, "lim_cpu": 2, "lim_mem": 2}
+    """
     try:
         max_cpu = 0
         max_memory = 0
```

Comment:
It's a bit misleading to call these `max_cpu` and `max_memory`, since they are only used as constant zero values it seems.

Reply:
removed `max_` prefix in few places :+1: